build.sbt

```diff
@@ -140,7 +141,8 @@ lazy val rootJVM3 = project
     ideSkipProject     := true
   )
   .aggregate({
-    val excluded: Set[ProjectReference] = Set(clientJS, clientNative, clientLaminext, codegenSbt, akkaHttp)
+    val excluded: Set[ProjectReference] =
```

Comment:
I'm hoping one day we'll have Scala 3 as the base version for the repo. 

If it's something that's quick to add, could we also have this running for Scala 3 just to reduce migration efforts later on? If it takes more than 10 mins to setup then no worries we can do it at a later time

Reply:
I'll try to enable it, it's a somewhat unique module because it's getting wrapped into a fat jar so I need to make sure that it doesn't get confused about how to do that. I was actually thinking this would have been good practice for writing something in scala3 so I might go back through in a later PR and update the source too.

Reply:
Looks like enabling it works